Cargo.toml

```
@@ -19,7 +19,7 @@ path = "src/lib.rs"
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
 bitcoin = { version = "0.31.0", features = ["serde", "std"], default-features = false }
-hex = { package = "hex-conservative", version = "*" }
+hex = { package = "hex-conservative", version = "0.2.0" }
```

```suggestion
hex = { package = "hex-conservative", version = "0.2" }
```

We could use this. It has the same semantic versioning meaning but with 2 less characters.